Update repo caching (#4684)
dbt-labs/docs.getdbt.com: website/snippets/_cloud-environments-info.md

dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 

Below lists the situations when dbt Cloud uses the cached copy:

- Git authentication fails.
- There are syntax errors in the `packages.yml` file. To catch these errors sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  
- A package is incompatible with the dbt version being used. To catch this incompatibility sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  

> 

Reviewer: For completeness, don't we also want to include failures of 3rd party services (github, dbt package hub, etc.) in this section?

Author: great flag! i agree with you. will update shortly

dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 

> Below lists the situations when dbt Cloud uses the cached copy:

Reviewer 2: i've read and been told that we should avoid using directional language like below or above for accessibility reasons
```suggestion
dbt Cloud will use the cached copy of your project's Git repo under these circumstances:
```

Author: good flag!

> - Outages from third-party services (for example, your Git provider).

Reviewer 2: adding a non git related service to be extra explicit

```suggestion
- Outages from third-party services (for example, the [dbt package hub](https://hub.getdbt.com/)).
```

> - Outages from third-party services (for example, the [dbt package hub](https://hub.getdbt.com/)).

Reviewer 2: i think it is helpful to have another example, on top of your git one. so adding the git bc that's def a big example that would be helpful for users to know

```suggestion
- Outages from third-party services (for example,  your Git provider or the [dbt package hub](https://hub.getdbt.com/)).
```

Author: the Git example is actually mentioned in the screenshot so we're good there

- Git authentication fails.

> - There are syntax errors in the `packages.yml` file. To catch these errors sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  

Reviewer 2: ```suggestion
- There are syntax errors in the `packages.yml` file. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to find these errors earlier.
```

> - There are syntax errors in the `packages.yml` file. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to find these errors earlier.

Author: ```suggestion
- There are syntax errors in the `packages.yml` file. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to find these errors sooner.
```

> - A package is incompatible with the dbt version being used. To catch this incompatibility sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  

Reviewer 2: sugg active voice
```suggestion
- If a package doesn't work with the current dbt version. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to identify this issue sooner.
```
